packages/app-mobile/components/NoteEditor/ImageEditor/js-draw/applyTemplateToEditor.ts

```diff
@@ -18,9 +18,7 @@ const applyTemplateToEditor = async (editor: Editor, templateData: string) => {
 				'zIndex': 0,
 				'data': {
 					'mainColor': '#ffffff',
-					'backgroundType': BackgroundComponentBackgroundType.Grid,
-					'gridSize': 25,
-					'gridStrokeWidth': 0.7,
+					'backgroundType': BackgroundComponentBackgroundType.SolidColor,
```

Comment:
is the default now solidcolor instead of grid?

Reply:
Yes. See "The grid shouldn't be displayed in the note viewer" section in the above comment.